src/Microsoft.Health.Fhir.Liquid.Converter/Processors/FhirProcessor.cs

```diff
@@ -0,0 +1,17 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using Microsoft.Health.Fhir.Liquid.Converter.Models;
+
+namespace Microsoft.Health.Fhir.Liquid.Converter.Processors
+{
+    public class FhirProcessor : JsonProcessor
```

Comment:
is it an inheritance? do we have other option?

Reply:
After offline discussion, we added doc here for refinement Todo list.